pkg/build/api/helpers.go

```diff
@@ -17,3 +17,32 @@ func BuildToPodLogOptions(opts *BuildLogOptions) *kapi.PodLogOptions {
 		LimitBytes:   opts.LimitBytes,
 	}
 }
+
+// FilterBuilds returns array of builds that satisfies predicate function.
+func FilterBuilds(builds []Build, predicate func(Build) bool) []Build {
+	if len(builds) == 0 {
+		return builds
+	}
+
+	result := make([]Build, 0)
+	for _, build := range builds {
+		if predicate(build) {
+			result = append(result, build)
+		}
+	}
+
+	return result
+}
+
+// ByBuildConfigLabelPredicate matches all builds that have build config label with specified value.
+func ByBuildConfigLabelPredicate(labelValue string) func(Build) bool {
```

Comment:
>in fact i'd probably call it "HasBuildConfigLabelFilter"

@bparees The initial idea was to make it looks like a single readable sentence when it's used with `FilterBuilds`:

```FilterBuilds(Items, ByBuildConfigLabelPredicate(name))```

But because of imports now it looks like this:

```buildapi.FilterBuilds(buildList.Items, buildapi.ByBuildConfigLabelPredicate(name))```

Hm... Do you think I should change it? (I don't want to argue but just want to share my thoughts and intentions that I had. The last word, of course, by you and other team members.)